packages/backend/src/modules/tracked-txs/modules/liveness/api/LivenessController.ts

```diff
@@ -167,6 +160,17 @@ export class LivenessController {
           {},
         )
 
+      if (project.livenessConfig) {
+        const { from, to } = project.livenessConfig.duplicateData
+
+        withSyncedUntil[to] = {
+          ...withSyncedUntil[from],
+        } as LivenessApiProject[
+          | 'batchSubmissions'
+          | 'proofSubmissions'
+          | 'stateUpdates']
+      }
```

Comment:
This looks a bit better than the 'as'
```suggestion
     if (project.livenessConfig) {
        const { from, to } = project.livenessConfig.duplicateData
        const data = withSyncedUntil[from]
        assert(data, 'From data must exist')
        withSyncedUntil[to] = { ...data }
      }
```